config/config-api/src/com/thoughtworks/go/config/merge/MergeEnvironmentConfig.java

```diff
@@ -414,7 +414,7 @@ public boolean hasVariable(String variableName) {
 
     @Override
     public boolean hasSamePipelinesAs(EnvironmentConfig other) {
-        return false;
+        return false; //TODO: jyoti - is this correct?
```

Comment:
Spoke to @tomzo. He would be updating the implementation of this method as a part of #2266